app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -241,13 +241,15 @@ def track_events(otp_delivery_preference:, otp_delivery_selection_result:)
           reauthentication: true,
           phone_number: parsed_phone.e164,
           otp_delivery_method: otp_delivery_preference,
+          failure_reason: @telephony_result.errors,
```

Comment:
In other places I've seen us use [`parse_failure_reason`](https://github.com/18F/identity-idp/blob/393294ac6a704e49ca770b31fa3076e60db4c37e/app/services/irs_attempts_api/tracker.rb#L63-L65). Would we expect to use it here?

Reply:
Excellent comment!  The `telephony_result.errors` was not constructed to match other error structures.  I'll double-check to see if it would come through correctly, if we use `parse_failure_reason`...

Reply:
It seems that it works!  Change has been made - thank you!